tests/README.md

```diff
@@ -20,3 +20,29 @@ https://github.com/vmg/clar
 * Make sure everything is fine.
 
 * Send your pull request. That's it.
+
+
+Memory leak checks
+------------------
+
+These are automatically run as part of CI, but if you want to check locally:
+
+#### Linux
+
+Uses [`valgrind`](http://www.valgrind.org/):
+
+```console
+$ cmake -DBUILD_CLAR=ON -DVALGRIND=ON ..
+$ cmake --build .
+$ valgrind --leak-check=full --show-reachable=yes --num-callers=50 --suppressions=../libgit2_clar.supp \
+  ./libgit2_clar
+```
+
+#### OSX
```

Comment:
`macOS`, but I'd be fine with `System 6` 😜.

Reply:
Indeed, let's use `macOS` here.  @rcoup if you wouldn't mind pushing an update, we'll :shipit: .